src/components/Footer/Footer.js

```diff
@@ -403,9 +348,7 @@ Footer.propTypes = {
   isSelf: PropTypes.bool,
   disableComment: PropTypes.bool,
   blockStatus: PropTypes.any,
-  loadingVote: PropTypes.any,
-  showScoreButton: PropTypes.bool,
-  onPressScore: PropTypes.func
+  loadingVote: PropTypes.any
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The props `onPressScore` and `showScoreButton` have been removed from the PropTypes validation. This is consistent with the changes made in the component's props.
```diff
-  showScoreButton: PropTypes.bool,
-  onPressScore: PropTypes.func
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->